karma.conf.js

```diff
@@ -0,0 +1,29 @@
+const puppeteer = require("puppeteer");
+process.env.CHROME_BIN = puppeteer.executablePath();
```

Comment:
Why is this necessary? Can't we just ask for a Chrome instance:

https://github.com/unexpectedjs/unexpected/blob/master/.travis.yml#L34-L35

Reply:
It would be nice to remove puppeteer.

Reply:
Oh wow, just saw the Chrome addon thing and had no idea you could ask for it! Let's do that :-)